### PR TITLE
Changed-Icon-file-name-for-hmmm

### PR DESCRIPTION
**Pull request by Yahmooo**
### 📝 Description

changed Icon name for Hmmm token.
Contract address (bsc)-
Hmmm - 0x88Cf50037f8069071Cd56f641a81b369a5Fd3a42

### ❓ Context

- **Impacted projects**: `` @ledgerhq/live-common
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: .changeset/lazy-pumpkins-visit.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": patch
+---
+
+changed the files name
```
Comment (nparigi-ledger) [inline .changeset/lazy-pumpkins-visit.md:5]: Hello Yahmooo :vulcan_salute: 

Thanks for your PR !
The comment on this changeset is really not explicit.
Can you explain that it is a crypto icon that you changed, and which one?

**Review by nparigi-ledger — CHANGES_REQUESTED** [review]

File: .changeset/lazy-pumpkins-visit.md (hunk 1 above)
Comment (Yahmooo) [reply .changeset/lazy-pumpkins-visit.md]: Changed it!

**Review by nparigi-ledger — APPROVED** [review]

**Comment by nparigi-ledger** [comment]
Approved :ok_hand:

**Comment by Yahmooo** [comment]
> Approved 👌

Thanks! Can I do something about these errors?

**Pull request merged**